### PR TITLE
Show Status in order view for user

### PR DESCRIPTION
**Pull request by WilRyan**

### CODE REVIEW

File: collivery.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * Plugin Name: MDS Collivery
  * Plugin URI: https://collivery.net/integration/woocommerce
  * Description: Plugin to add support for MDS Collivery in WooCommerce.
```
Comment (brycelarge) [inline collivery.php]: Im sure this version needed to match the version above, cant remember but have a look back at some of my commits for version change

File: collivery.php [hunk 2]
```diff
@@ -183,4 +183,15 @@ function mds_change_default_checkout_country()
...
         add_filter('default_checkout_country', 'mds_change_default_checkout_country');
     }
+    
+    add_action( 'woocommerce_view_order', 'before_woocommerce_order_details', 5 );
+    function before_woocommerce_order_details($order_id){
+        global $wpdb;
+        $table_name = $wpdb->prefix.'mds_collivery_processed';
+        $data_ = $wpdb->get_results('SELECT * FROM `'.$table_name.'` WHERE order_id='.$order_id.';', OBJECT);
+        $mds = MdsColliveryService::getInstance();
+        $waybillId = (int)$data_[0]->waybill;
```
Comment (brycelarge) [inline collivery.php]: This might cause the customer to be throttled, not sure what the limits are

Comment (WilRyan) [reply collivery.php]: You're right, each time the page load its a request :|

File: collivery.php [hunk 3]
```diff
@@ -183,4 +183,15 @@ function mds_change_default_checkout_country()
...
+    
+    add_action( 'woocommerce_view_order', 'before_woocommerce_order_details', 5 );
+    function before_woocommerce_order_details($order_id){
+        global $wpdb;
+        $table_name = $wpdb->prefix.'mds_collivery_processed';
+        $data_ = $wpdb->get_results('SELECT * FROM `'.$table_name.'` WHERE order_id='.$order_id.';', OBJECT);
+        $mds = MdsColliveryService::getInstance();
+        $waybillId = (int)$data_[0]->waybill;       
+        $statusText = $mds->returnColliveryClass()->getStatus($waybillId)? $mds->returnColliveryClass()->getStatus($waybillId)['status_text']: "No Status";
+        echo "Shipping Status (MDS Collivery ".$waybillId.") : ".$statusText;
```
Comment (peterchaula) [inline collivery.php]: Just put the variables inside the string since you are using double quotes

Comment (WilRyan) [reply collivery.php]: no this PR :)

**Pull request closed without merging**